doc/source/conf.py

```diff
@@ -316,3 +384,8 @@ def setup(app):
 
     # Julia lexer
     app.add_lexer("julia", JuliaLexer)
+
+    return {
+        "parallel_read_safe": True,
+        "parallel_write_safe": True,
+    }
```

Comment:
```suggestion
```

Reply:
We don't need to return this information. It's only necessary when building a sphinx extension.